src/commontools.c

```diff
@@ -107,20 +108,18 @@ btc_bool hd_print_node(const btc_chainparams* chain, const char* nodeser)
     if (!btc_hdnode_deserialize(nodeser, chain, &node))
         return false;
 
-    size_t strsize = 128;
     char str[128];
+    size_t strsize = sizeof(str);
```

Comment:
This is a solution, too, ofc. Thanks for helping out here, @xloem!

Reply:
I bump into a lot of bugs so sometimes I really notice places where somebody could mess stuff up by changing one thing and not another.  Thanks addressing these things so long ago.